packages/@aws-cdk/aws-codecommit/lib/repository.ts

```diff
@@ -176,6 +176,7 @@ export interface IRepository extends IResource, notifications.INotificationRuleS
   ): notifications.INotificationRule;
 
   /**
+   * @deprecated
    * Defines a CodeStar Notification rule which triggers when a pull request is merged.
    */
```

Comment:
We need to state what is the replacement. Also, `@deprecated` comes last, not first.

```suggestion
  /**
   * Defines a CodeStar Notification rule which triggers when a pull request is merged.
   * @deprecated this method has a typo in its name, use notifyOnPullRequestMerged instead
   */
```

Reply:
ah thanks, wasn't quite sure how to format that!